docs/ja/about-appium/getting-started.md

```diff
@@ -0,0 +1,274 @@
+
+先に進む前に、お好きな言語でクライアントをダウンロードし、
+準備ができていることを確認してください。
+
+### Appium の開始
+
+(NPMのインストールが成功したと仮定して) 
+これで、以下のようにコマンドラインから実行することで、Appium サーバーをキックアップすることができます。
+
+```
+appium
+```
+
+または、Appium Desktop 内の巨大な Start Server ボタンをクリックしてください。
+
+Appium は、実行している Appium のバージョンとどのポートを待ち受けているかを示す小さなウェルカムメッセージを表示します
+(デフォルトは `4723` です)。
+このポートで Appium に接続するようにテストクライアントを指示する必要があるので、
+このポート情報は重要です。
+ポートを変更したい場合は、
+Appium を起動する際に `-p` フラグを使用して変更することができます
+([サーバパラメータ](/docs/en/writing-running-appium/server-args.md)の完全なリストをチェックしてください)。
+
+### 最初のテストの実行
+
+このセクションでは、基本的な "Hello World" の Android テストを実行します。
+Android を選択したのは、すべてのプラットフォームで利用可能だからです。
+[UiAutomator2 ドライバ](/docs/en/drivers/android-uiautomator2.md)を使用しますので、
+そのドキュメントを読み、システムを適切に設定してください。
+また、言語として JavaScript を使用しますので、
+追加の依存関係に対処する必要はありません。
+
+(Android 以外のものを
+JavaScript 以外のものを使って自動化したいと思うこともあるでしょう。
+その場合は、多くの言語やプラットフォームに対応した
+[コードサンプル](https://github.com/appium/appium/tree/master/sample-code)を用意しています。)
+
+#### 前提条件
+
+- あなたが Android 8.0 エミュレータを設定して実行していると仮定します
+  (例は、低バージョンで動作します。
+  ちょうどそれに応じてバージョン番号を修正します)
+- あなたが[このテスト APK](https://github.com/appium/appium/raw/master/sample-code/apps/ApiDemos-debug.apk) をダウンロードして、
+  ローカルファイルシステム上で
+  利用可能であると仮定します
+
+#### Appium クライアントの設定
+
+この例では、Appium クライアントとして [Webdriver.io](http://webdriver.io) を使用します。
+この例のディレクトリを作成し、実行します。
+
+```
+npm init -y
+```
+プロジェクトを初期化したら、`webdriverio` をインストールします:
+```
+npm install webdriverio
+```
+
+#### セッションの初期化
+
+ここで `index.js` という名前のテストファイルを作成し、
+クライアントオブジェクトを初期化します:
+
+```js
+// javascript
+const wdio = require("webdriverio");
+```
+
+次にやるべきことは、Appium セッションを開始することです。
+これを行うには、サーバオプションと Desired Capabilities のセットを定義し、
+それらを使って `wdio.remote()` を呼び出します。
+Desired Capabilities は、セッションの初期化中に Appium サーバーに送信されるキーと値のセットで、
+どのようなことを自動化したいかを Appium に伝えます。
+どのような Appium ドライバでも、必要な能力の最小セットには以下のものが含まれているはずです。
+
+- `platformName`: 自動化するプラットフォームの名前
+- `platformVersion`: 自動化するプラットフォームのバージョン
+- `deviceName`: 自動化するデバイスの種類
+- `app`: 自動化したいアプリへのパス (ただし、Web ブラウザを自動化する場合は代わりに
+  `browserName` capability を使用します)
+- `automationName`: 使用するドライバーの名前
+
+Desired Capabilities の詳細と、
+Appium で使用できるすべての機能のリストについては、
+[Capabilities doc](/docs/en/writing-running-appium/caps.md)を参照してください。
+
+ここでは、テストファイルでセッションを構築する方法を説明します:
+
+```js
+// javascript
+const opts = {
+  path: '/wd/hub',
+  port: 4723,
+  capabilities: {
+    platformName: "Android",
+    platformVersion: "8",
+    deviceName: "Android Emulator",
+    app: "/path/to/the/downloaded/ApiDemos-debug.apk",
+    appPackage: "io.appium.android.apis",
+    appActivity: ".view.TextFields",
+    automationName: "UiAutomator2"
+  }
+};
+
+async function main () {
+  const client = await wdio.remote(opts);
+
+  await client.deleteSession();
+}
+
+main();
+```
+
+#### テストコマンドの実行
+
+Appium ポートを指定し、要件に一致するように Desired Capabilities を構築したことがわかります
+(ただし、パスをあなたのシステムの実際のダウンロードパスに置き換えることを忘れないでください)。
+この事実を `webdriverio` に登録して、
+Appium サーバへの接続を表すクライアントオブジェクトができました。
```

Comment:
```suggestion
Appium サーバーへの接続を表すクライアントオブジェクトができました。
```

It would be great to define words consistency since you already appended "ー" for サーバー in some lines